Unit tests failing for swap and custom-assets

**Describe the bug**
The cargo test command is failing on the develop branch due to an error in unit testing in pallets uniswap and custom-assets.

**To Reproduce**
Steps to reproduce the behavior:
1. Cargo test on develop branch

**Expected behavior**
Expected all tests to pass

**Screenshots**
Not applicable

**Additional context**
Error Logs
```
error[E0277]: the trait bound `mock::Test: pallet_idenity::Trait` is not satisfied
  --> pallets/polkadex-uniswap/src/mock.rs:54:6
   |
54 | impl Trait for Test {
   |      ^^^^^ the trait `pallet_idenity::Trait` is not implemented for `mock::Test`

error[E0277]: the trait bound `mock::Test: pallet_idenity::Trait` is not satisfied
  --> pallets/polkadex-custom-assets/src/mock.rs:76:6
   |
76 | impl Trait for Test {
   |      ^^^^^ the trait `pallet_idenity::Trait` is not implemented for `mock::Test`
   |
  ::: pallets/polkadex-custom-assets/src/lib.rs:31:40
   |
31 | pub trait Trait: frame_system::Trait + pallet_idenity::Trait {
   |                                        --------------------- required by this bound in `Trait`

error: could not compile `pallet-idenity`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error[E0277]: the trait bound `mock::Test: pallet_idenity::Trait` is not satisfied
  --> pallets/polkadex-uniswap/src/mock.rs:63:6
   |
63 | impl polkadex_custom_assets::Trait for Test{
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `pallet_idenity::Trait` is not implemented for `mock::Test`
   |
  ::: /mnt/c/Users/Gautham/Desktop/Work/Polkadex/pallets/polkadex-custom-assets/src/lib.rs:31:40
   |
31 | pub trait Trait: frame_system::Trait + pallet_idenity::Trait {
   |                                        --------------------- required by this bound in `polkadex_custom_assets::Trait`

error[E0277]: the trait bound `mock::TestRuntime: pallet_idenity::Trait` is not satisfied
  --> pallets/polkadex-engine/src/mock.rs:57:6
   |
57 | impl Trait for TestRuntime {
   |      ^^^^^ the trait `pallet_idenity::Trait` is not implemented for `mock::TestRuntime`

error[E0277]: the trait bound `mock::TestRuntime: pallet_idenity::Trait` is not satisfied
  --> pallets/polkadex-engine/src/mock.rs:69:6
   |
69 | impl polkadex_custom_assets::Trait for TestRuntime {
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `pallet_idenity::Trait` is not implemented for `mock::TestRuntime`
   |
  ::: /mnt/c/Users/Gautham/Desktop/Work/Polkadex/pallets/polkadex-custom-assets/src/lib.rs:31:40
   |
31 | pub trait Trait: frame_system::Trait + pallet_idenity::Trait {
   |                                        --------------------- required by this bound in `polkadex_custom_assets::Trait`

error[E0277]: the trait bound `mock::TestRuntime: pallet_idenity::Trait` is not satisfied
  --> pallets/polkadex-engine/src/mock.rs:79:6
   |
79 | impl polkadex_swap_engine::Trait for TestRuntime {
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `pallet_idenity::Trait` is not implemented for `mock::TestRuntime`
   
```

--- a/pallets/polkadex-custom-assets/src/mock.rs
+++ b/pallets/polkadex-custom-assets/src/mock.rs
@@ -69,6 +69,19 @@ impl AssetIdProvider for Test {
 }
 
 parameter_types! {
+    pub const MaxSubAccounts: u32 = 10;
+    pub const MaxRegistrars: u32 = 10;
+}
+
+impl pallet_idenity::Trait for Test {
+    type Event = ();
+    type MaxSubAccounts = MaxSubAccounts;
+    type MaxRegistrars= MaxRegistrars;
+
+
+}
+
+parameter_types! {
 pub const maxLocks: u32 = 10;
 pub const existentialDeposit: u128 = 1;
 }

--- a/pallets/polkadex-uniswap/Cargo.toml
+++ b/pallets/polkadex-uniswap/Cargo.toml
@@ -20,6 +20,7 @@ frame-system = { default-features = false, version = '2.0.0' }
 sp-runtime = { default-features = false, version = '2.0.0' }
 sp-arithmetic = { default-features = false, version = '2.0.0' }
 sp-core = { default-features = false, version = '2.0.0' }
+pallet-idenity = {default-features = false, path = "../polkadex-identity-pallet"}
 polkadex-custom-assets = { path = "../polkadex-custom-assets", default-features = false}
 [dev-dependencies]
 sp-io = { default-features = false, version = '2.0.0' }
@@ -31,4 +32,5 @@ std = [
     'codec/std',
     'frame-support/std',
     'frame-system/std',
+    'pallet-idenity/std'
 ]

--- a/pallets/polkadex-uniswap/src/lib.rs
+++ b/pallets/polkadex-uniswap/src/lib.rs
@@ -16,6 +16,7 @@ use sp_std::vec;
 use sp_runtime::traits::{AccountIdConversion, Saturating, Zero};
 
 
+
 #[cfg(test)]
 mod mock;
 

--- a/pallets/polkadex-uniswap/src/mock.rs
+++ b/pallets/polkadex-uniswap/src/mock.rs
@@ -5,6 +5,7 @@ use sp_runtime::{traits::{BlakeTwo256, IdentityLookup}, testing::Header, Perbill
 use frame_system as system;
 
 
+
 impl_outer_origin! {
 	pub enum Origin for Test {}
 }
@@ -51,9 +52,24 @@ impl system::Trait for Test {
 parameter_types! {
     pub const TradingPathLimit: usize = 10;
 }
+
+
 impl Trait for Test {
     type Event = ();
     type TradingPathLimit = TradingPathLimit;
+}
+
+parameter_types! {
+    pub const MaxSubAccounts: u32 = 10;
+    pub const MaxRegistrars: u32 = 10;
+}
+
+impl pallet_idenity::Trait for Test {
+    type Event = ();
+    type MaxSubAccounts = MaxSubAccounts;
+    type MaxRegistrars= MaxRegistrars;
+
+
 }
 parameter_types! {
     pub const MaxLocks: u32 = 10;

--- a/pallets/polkadex-uniswap/src/tests.rs
+++ b/pallets/polkadex-uniswap/src/tests.rs
@@ -336,7 +336,7 @@ pub fn test_orderbook_functions () {
 
         // Check for best deal and Order execution
         // Argument Provided :- max-supply:- 1*UNIT and min-target 1*UNIT -> It should return false
-        assert!(!PolkadexSwapEngine::swap_by_orderbook(&alice, (second_asset_id, first_asset_id), true, PolkadexSwapEngine::convert_balance_to_fixedU128(1 * UNIT).unwrap(), PolkadexSwapEngine::convert_balance_to_fixedU128(UNIT).unwrap(), ));
+        assert!(!PolkadexSwapEngine::swap_by_orderbook(&alice, (second_asset_id, first_asset_id), true, PolkadexSwapEngine::convert_balance_to_fixedU128(1 * UNIT).unwrap(), PolkadexSwapEngine::convert_balance_to_fixedU128(UNIT).unwrap(), second_asset_id));
         // Check for best case
         // Check for free balance
         assert_eq!(polkadex_custom_assets::Module::<Test>::free_balance(&alice, first_asset_id), 8*UNIT);

